src/frontend/src/reducers/neighborhoods/selectors/neighborhoodsSortedWithRoutes.ts

```diff
@@ -0,0 +1,148 @@
+// Refactored from old codebase neighborhoods-sorted-with-routes at commit 6f17e33
+
+import concat from "lodash/concat";
+import findIndex from "lodash/findIndex";
+import get from "lodash/get";
+import includes from "lodash/includes";
+import orderBy from "lodash/orderBy";
+import pullAt from "lodash/pullAt";
+
+import {
+    BOOST_DOWNTOWN_RESULT_PLACE,
+    DOWNTOWN_AREAS,
+    MAX_TRAVEL_TIME,
+    RESULTS_WITH_DOWNTOWN,
+} from "../../../constants";
+import { createSelector } from "@reduxjs/toolkit";
+
+import type { Feature, Point } from "geojson";
+import type { NeighborhoodProperties } from "../types";
+import type { RootState } from "store/store";
+
+import selectNeighborhoodRoutes from "../../networks/selectors/networkNeighborhoodRoutes";
+import selectNeighborhoodTravelTimes from "./allNeighborhoodTravelTimes";
+import importanceCriteriaScoreWeights from "reducers/userProfile/selectors/importanceCriteriaScoreWeights";
+import { createNeighborhoodWeightedScore } from "../utils/createNeighborhoodWeightedScore";
+import { selectAllNetworksDataReady } from "src/reducers/networks/networksSlice";
+
+const getZipCodeListFromNeighborhoods = (
+    neighborhoods: Feature<Point, NeighborhoodProperties>[]
+) => neighborhoods.map(n => n.properties.id);
+
+export default createSelector(
+    [
+        selectNeighborhoodRoutes,
+        selectNeighborhoodTravelTimes,
+        importanceCriteriaScoreWeights,
+        selectAllNetworksDataReady,
+        (state: RootState) => get(state, "neighborhoods.neighborhoods"),
+        (state: RootState) => get(state, "networks.activeMode"),
+    ],
+    (
+        neighborhoodRoutes,
+        travelTimes,
+        userScoreWeights,
+        networksReady,
+        neighborhoods,
+        activeNetworkMode
+    ) => {
+        const rankedLists: { recommended: string[]; tooFar: string[] } = {
+            recommended: [],
+            tooFar: [],
+        };
+
+        if (!networksReady) {
+            return rankedLists;
+        }
+
+        const recommendedNeighborhoodsList: Feature<
+            Point,
+            NeighborhoodProperties
+        >[] = [];
+        const tooFarNeighborhoodsList: Feature<
+            Point,
+            NeighborhoodProperties
+        >[] = [];
+
+        const useTransit = activeNetworkMode === "car";
```

Comment:
I am a bit confused by this line. Why is `useTransit` true if the active mode is car?